src/controllers/accountAdder/accountAdder.ts

```diff
@@ -864,8 +866,12 @@ export class AccountAdderController extends EventEmitter {
               factoryAddr,
               bytecode,
               salt
+            },
+            preferences: {
+              label: DEFAULT_ACCOUNT_LABEL,
+              pfp: addr
             }
-          },
+          } as Account,
```

Comment:
I see no need for casting to `Account`, the TypeScript on my end doesn't complain about this type. Please double-check on your end?